src/types.ts

```diff
@@ -159,6 +197,10 @@ export interface AllTimeStakingStats {
     totalRewardsPaidInEth: number;
 }
 
+export interface StakingPoolResponse {
+    poolId: string;
+    stats: PoolWithHistoricalStats;
```

Comment:
should we call it "pool" something? it is a pool just with more info .

Reply:
How about `stakingPool` to be consistent with `stakingPools` returned from the pools endpoint?